mypyc/ir/rtypes.py

```diff
@@ -436,6 +445,123 @@ def deserialize(cls, data: JsonDict, ctx: 'DeserMaps') -> 'RTuple':
 )
 
 
+def compute_rtype_alignment(typ: RType) -> int:
+    """Compute alignment of a given type based on platform alignment rule"""
+    platform_alignment = 4 if IS_32_BIT_PLATFORM else 8
```

Comment:
`4 if IS_32_BIT_PLATFORM else 8` gets repeated a few times, so it's better to define it as a named constant.